Codecademy/gamut
# feat(SkillPathCard): New skill path card

### Overview


This PR adds the existing [SkillPathCard](https://github.com/codecademy-engineering/mono/blob/develop/apps/portal-app/src/components/CatalogPage/SkillPathCard.tsx) component into gamut-labs. 

Related gamut PRs for context 
https://github.com/Codecademy/gamut/pull/2653
https://github.com/Codecademy/gamut/pull/2656



### PR Checklist

- [x] Related to designs: https://www.figma.com/file/q0bRIpoO92YfCpy4nCiQGI/What's-your-goal---iteration?node-id=702%3A3308&t=GjAsG8u5UGR3D1jO-0
- [x] Related to JIRA ticket: [DISC-1042]

#### Testing Instructions

1. Open the latest Styleguide deploy preview below and search for SkillPathCard
2. Review this page and make sure everything is captured for this new skill path card 
3. Open the monorepo PR env link and see that the skill path cards on catalog, hubs, and full catalog page look the same as production. 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34458) 
| Monorepo       | [Portal PR](https://github.com/codecademy-engineering/mono/pull/1136)  | [Monorepo Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1136)   


[DISC-1042]: https://codecademy.atlassian.net/browse/DISC-1042?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
